packages/design-system/src/styles/settings/variables/_color.scss

```diff
@@ -1,114 +1,83 @@
-// Generic
-$color-white: #fff !default;
```

Comment:
I don't think this was supposed to be commited, was it? I don't see anywhere in the tokens package where it would modify other package's files. And the goal is to have the core module import something from `/dist`, right?

Reply:
This was only committed to build the test site for Scott to compare with the old site as the colors have changed with his system, I will be backing this out before merge, forgot to note!